src/app/cluster/details/external-cluster/external-machine-deployment-details/component.ts

```diff
@@ -32,6 +31,7 @@ import {MemberUtils, Permission} from '@shared/utils/member';
 import {forkJoin, Subject, timer} from 'rxjs';
 import {switchMap, take, takeUntil} from 'rxjs/operators';
 import {ExternalMachineDeploymentService} from '@core/services/external-machine-deployment';
+import {UpdateExternalClusterMachineDeploymentDialogComponent} from '@app/cluster/details/external-cluster/update-external-cluster-machine-deployment-dialog/component';
```

Comment:
I think it would be better to use relative import here (and external-machine-deployments-list component below).